tcms/telemetry/api.py

```diff
@@ -41,5 +41,21 @@ def breakdown(query=None):
 
 
 def _get_field_count_map(test_cases, expression, field):
-    query_set = test_cases.values(field).annotate(count=Count(expression))
-    return [[entry[field], entry['count']] for entry in query_set]
+    query_set_confirmed = test_cases.filter(
+        case_status__name='CONFIRMED'
```

Comment:
Use `TestCaseStatus.get_confirmed()` to get the confirmed object PK and also make that method override the language to English when fetching the object from the DB. See `TestCaseStatus.is_confirmed`.